### PR TITLE
feat(hover-card): create hover-card primitive

### LINKED ISSUE #166: [Primitive] Hover card
https://www.radix-ui.com/primitives/docs/components/hover-card

### PR DESCRIPTION
**Pull request by pawel-twardziak**
Closes #166

### CODE REVIEW

**Comment by pawel-twardziak** [comment]
@pimenovoleg @artembelik @mrfrac I guess it's ready, please, look at this :)

**Comment by pawel-twardziak** [comment]
https://github.com/user-attachments/assets/f4bc48a0-49ab-497a-8f1c-514756534447

File: packages/primitives/hover-card/src/hover-card-close.directive.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { Directive, effect, ElementRef, forwardRef, inject, Renderer2, untracked } from '@angular/core';
+import { RdxHoverCardCloseToken } from './hover-card-close.token';
+import { injectHoverCardRoot } from './hover-card-root.inject';
+
+/**
+ * TODO: to be removed? But it seems to be useful when controlled from outside
+ */
+@Directive({
```
Comment (pimenovoleg) [inline packages/primitives/hover-card/src/hover-card-close.directive.ts:8]: Let's leave it for now, wait, and see.

**Review by pimenovoleg — APPROVED** [review]

**Comment by pimenovoleg** [comment]
This looks great! I really like how most of the logic is being reused here

**Comment by pawel-twardziak** [comment]
> This looks great! I really like how most of the logic is being reused here

yes, imho it's around 95% of the logic that is gonna be reused next :) I have a plan for it 🤖

**Pull request merged**